src/packages/ganache/npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ganache",
-	"version": "7.0.0-alpha.2",
```

Comment:
I think you need to also commit this file, as it still contains a `ttypescript` reference, but I don't think it should. Can you check?